go/vt/vtgate/vtgate.go

```diff
@@ -216,6 +216,23 @@ var (
 		[]string{"Operation", "Keyspace", "DbType"})
 )
 
+func ClearMetrics() {
+	vschemaCounters.ResetAll()
+	errorCounts.ResetAll()
+	warnings.ResetAll()
+	vstreamSkewDelayCount.Reset()
+	vindexUnknownParams.Reset()
+	timings.Reset()
+	rowsReturned.ResetAll()
+	rowsAffected.ResetAll()
+	queriesProcessed.ResetAll()
+	queriesRouted.ResetAll()
+	queriesProcessedByTable.ResetAll()
+	queriesRoutedByTable.ResetAll()
+	vttabletTimings.Reset()
+	tabletCallErrorCount.ResetAll()
+}
```

Comment:
This will be a problem because 1) this does not include the full list of vtgate stats today 2) when ones are added in the future it will not be clear (or expected) to have to find this and add the new metric here for the reset.